MQClient/queue.py

```diff
@@ -52,32 +60,45 @@ def prefetch(self, val: int) -> None:
                 del self.raw_sub_queue
 
     @property
-    def raw_pub_queue(self) -> typing.Any:
+    def raw_pub_queue(self) -> Pub:
+        """Get publisher queue."""
         if not self._pub_queue:
             self._pub_queue = self._backend.create_pub_queue(self._address, self._name)
+
+        if not self._pub_queue:
+            raise Exception("Pub queue failed to be created.")
         return self._pub_queue
```

Comment:
Something feels a little weird about a getter creating the publication queue.
I don't have an alternate design to offer off the top of my head, though.

Reply:
It's definitely one of those trade-offs. Considering the amount of work on the backend for opening a pub/sub, I think this is an effective pattern.